### PR TITLE
Add new code owner

### PR DESCRIPTION
**Pull request by ezodude**
## Description

Changes proposed in this pull request:

- Added @ezodude as a code owner.

## Related issue(s)

N/A

### CODE REVIEW

**Review by mszostok — APPROVED** [review]
LGTM, welcome to BotKube team! 🎉

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # BotKube maintainers (email: dev-team@botkube.io)
 
-* @mszostok @pkosiec @PrasadG193
+* @mszostok @pkosiec @PrasadG193 @ezodude
```
[the remark this answers is not in the document]
Comment (mszostok) [reply .github/CODEOWNERS:3]: yes 👍

[the remark this answers is not in the document]
Comment (ezodude) [reply .github/CODEOWNERS:3]: Just saw this. Will fix.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # BotKube maintainers (email: dev-team@botkube.io)
+# Maintainers are listed alphabetically
 
-* @mszostok @pkosiec @PrasadG193
+* @ezodude @mszostok @pkosiec @PrasadG193
```
Comment (pkosiec) [inline .github/CODEOWNERS:4]:
```suggestion
* @ezodude @mszostok @pkosiec @PrasadG193
```

**Review by pkosiec — APPROVED** [review]
Approved once again 🙂 Please remove the trailing space (you can apply the suggestion from GH UI) and merge 🚀

**Pull request merged**